examples/launch_plot_results.py

```diff
@@ -64,8 +64,13 @@
     tuner.run()
 
     tuning_experiment = load_experiment(tuner.name)
-    print(tuning_experiment)
 
+    # Print the best configuration found from experiment-results
     print(f"best result found: {tuning_experiment.best_config()}")
 
+    # Plot the best value found over time
     tuning_experiment.plot()
+
+    # Print the best configuration found from the tuner and retrain it
+    trial_id, best_config = tuner.best_config()
+    tuner.trial_backend.start_trial(config=best_config)
```

Comment:
Maybe plot again, and hopefully show improvement? Or consider splitting out into a separate retraining example? 

Otherwise it feels a bit random - why train again and then do nothing with it after?

Reply:
One use-case could be to run with a larger budget, I do not have one use-case personally but I know some people asks for this so it probably have an example showing how it can be done.